### ISSUE TITLE
Twig v2.7.3 breaks theme-bundle

### ISSUE THREAD

**Issue by alekseyp** [Sylius/Sylius#10253]
**Sylius version affected**: 1.4.1

**Description**  
```
!!  In Source.php line 30:
!!                                                                                 
!!    Argument 2 passed to Twig\Source::__construct() must be of the type string,  
!!     object given, called in /home/vagrant/Code/tps-sylius/vendor/sylius/theme-  
!!    bundle/src/Twig/ThemeFilesystemLoader.php on line 51                         
!!                                                                                 
!!  
```

**Steps to reproduce**  
Brand new install or `composer update`

**Possible Solution**  
v2.7.2 works fine.

**Comment by stefandoorn** [comment]
I'm experiencing the same issue with Twig `v2.7.4`.

**Comment by alekseyp** [comment]
2.7.4 worked well for me

**Comment by pamil** [comment]
@stefandoorn what version of ThemeBundle do you use?

**Comment by stefandoorn** [comment]
Sylius `v1.2.15`

**Comment by pamil** [comment]
This has been fixed for 1.3+ according to our release process: https://github.com/Sylius/SyliusThemeBundle/pull/22

**Comment by stefandoorn** [comment]
Ah, perfect!
